pom.xml

```diff
@@ -64,7 +64,7 @@
 
     <aerius-tools.version>1.1.1</aerius-tools.version>
     <spotless.version>2.5.0</spotless.version>
-    <jacoco.version>0.8.6</jacoco.version>
+    <jacoco.version>0.8.8</jacoco.version>
```

Comment:
might as well go to 0.8.12 right away? Higher java version support, and nothing breaking as far as I can tell.